src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx

```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
         });
         it('should set focus on the input field', () => {
-            setFocusOnInput();
-            expect(getInputNode()).toBe(document.activeElement);
+            setFocusOnInput(viewContainer);
+
+            expect(getInputNode(viewContainer)).toBe(document.activeElement);
         });
         it('should not set focus on the input field when the component is disabled', () => {
-            wrapper.setProps({ disabled: true });
-            setFocusOnInput();
-            expect(wrapper.find('input')).toHaveLength(0);
+            view.rerender(
+                <Combobox<{ name: string }>
+                    suggestions={suggestions}
+                    suggestionToString={suggestionToString}
+                    inputPlaceholder={inputPlaceholder}
+                    noSuggestionsPlaceholder={noSuggestionsPlaceholder}
+                    onSelectionAdd={mockOnSelectionAdd}
+                    onInputValueChange={mockOnInputValueChange}
+                    onBlur={mockOnBlur}
+                    disabled
+                />
+            );
+            setFocusOnInput(viewContainer);
+
+            expect(viewContainer.querySelectorAll('input')).toHaveLength(0);
         });
         it('should render noSuggestions placeholder when empty suggestions list is passed', () => {
-            wrapper.setProps({ suggestions: [] });
-            setFocusOnInput();
+            view.rerender(
+                <Combobox<{ name: string }>
+                    suggestions={[]}
+                    suggestionToString={suggestionToString}
+                    inputPlaceholder={inputPlaceholder}
+                    noSuggestionsPlaceholder={noSuggestionsPlaceholder}
+                    onSelectionAdd={mockOnSelectionAdd}
+                    onInputValueChange={mockOnInputValueChange}
+                    onBlur={mockOnBlur}
+                    disabled
+                />
+            );
+            setFocusOnInput(viewContainer);
+            // expect(viewContainer.querySelectorAll('li')).toHaveLength(1);
 
-            expect(wrapper.find('li')).toHaveLength(1);
-            expect(wrapper.find('li').childAt(0).text()).toEqual(noSuggestionsPlaceholder);
+            // expect(wrapper.find('li')).toHaveLength(1);
+            // expect(wrapper.find('li').childAt(0).text()).toEqual(noSuggestionsPlaceholder);
         });
         it('should render all suggestions from the list', () => {
-            setFocusOnInput();
+            setFocusOnInput(viewContainer);
 
-            expect(wrapper.find('li')).toHaveLength(suggestions.length);
+            expect(viewContainer.querySelectorAll('li')).toHaveLength(suggestions.length);
         });
         describe('when blurred', () => {
             it('should render selection placeholder', () => {
-                expect(wrapper.text()).toEqual(inputPlaceholder);
+                expect(screen.findByText('type here...')).toBeDefined();
```

Comment:
I saw in the docs `toBeInDocument`. Here you use `toBeDefined`. Can you explain the difference and why you choose to use this one?